packages/app/src/__deprecated__/main/functions/register-news-listener/get-updated-news.ts

```diff
@@ -9,34 +9,23 @@ import { NewsEntry } from "App/news/dto"
 import { Entry, EntryCollection } from "contentful"
 import fs from "fs-extra"
 import { normalizeContentfulData } from "App/news/helpers"
+import _ from "lodash"
```

Comment:
> - It seems like the import of a whole Lodash library will lead to the largest bundle size.
> - Less readable usage in the Javascript code.

[source](https://www.blazemeter.com/blog/import-lodash-libraries)

I suggest changing to  ⬇️ 
```suggestion
import { isEqual } from "lodash"
```